phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/HpoAnnotation.java

```diff
@@ -218,12 +239,16 @@ public Builder citations(List<String> cites) {
         this.citations=cites;
         return this;
       }
+      public Builder evidence(String evid) {
+        this.evidence = EvidenceCode.fromString(evid);
+        return this;
+      }
 
       public HpoAnnotation build() {
         if (citations==null){
           citations=ImmutableList.of();
```

Comment:
initialise this as an immutable list in the Builder fields - it is currently null there.